src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestRegistration/HomeTestRegistrationTableViewCell.swift

```diff
@@ -20,6 +20,12 @@ final class HomeTestRegistrationTableViewCell: UITableViewCell {
 
 		setup()
 	}
+	
+	override func layoutSubviews() {
+		super.layoutSubviews()
+		
+		gradientView.roundCorners(corners: [.topLeft, .topRight], radius: 16)
```

Comment:
Do we then need to reset the corners in init of gradientView for reuse?

Reply:
The cell is used once, without reuse.

Reply:
But this gradient view is used at various place in the app, by reseting in the init of GradientView we make sure that nothing it is expected.

Reply:
`roundCorners(corners:radius:)` is an extension of `UIView` ;)